### PR TITLE
[DEM] Moved files from the Python to the C++ part of the App

### PR DESCRIPTION
**Pull request by farrufat-cimne**
since PythonGenericFunctionUtility() was already updated

### CODE REVIEW

File: applications/DEMApplication/CMakeLists.txt [hunk 1]
```diff
@@ -98,6 +98,12 @@ set(KRATOS_DEM_APPLICATION_CORE
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/stationarity_checker.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/automatic_dt_process.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/multiaxial_control_module_generalized_2d_utilities.cpp
+
+    # processes
```
Comment (maceligueta) [inline applications/DEMApplication/CMakeLists.txt]: no need of this comment. We have been mixing all types. If yout put the comment, organize the rest too, but in my opinion it is not necessary

**Comment by maceligueta** [comment]
Maybe you need some KRATOS_API in that process which can't be linked...

**Comment by farrufat-cimne** [comment]
> Maybe you need some KRATOS_API in that process which can't be linked...

Seems to be fine in
CI / windows-without-unity (pull_request) Successful in 48m

**Comment by philbucher** [comment]
the non-unity CI only contains the Core as it is much slower
I will change the name

**Comment by philbucher** [comment]
Seems like you might be missing `KRATOS_API`

**Comment by farrufat-cimne** [comment]
Merged master to avoid conflict.
Does not compile until master is fixed.

**Review by philbucher — APPROVED** [review]
👍

**Pull request merged**